
Hi,
 The SAA9730 appears to be mips only, as it includes files
from asm/ which are only present in mips/ and mips64/
The patch below makes this only show up on the menu for mips32,

I didn't spot a define_bool CONFIG_MIPS64 or the likes, so
it probably needs improving if the driver works on mips64 too.

Also, the driver seems to support being built as a module, but
the config option was a bool.

regards,

Dave.

-- 
| Dave Jones.                    http://www.codemonkey.org.uk
| SuSE Labs .

diff -urN --exclude-from=/home/davej/.exclude linux-dj/drivers/net/Config.in linux-test/drivers/net/Config.in
--- linux-dj/drivers/net/Config.in	Mon Sep 24 00:36:27 2001
+++ linux-test/drivers/net/Config.in	Mon Sep 24 16:10:09 2001
@@ -191,8 +191,8 @@
       if [ "$CONFIG_OBSOLETE" = "y" ]; then
 	 dep_bool '    Zenith Z-Note support (EXPERIMENTAL)' CONFIG_ZNET $CONFIG_ISA
       fi
-      if [ "$CONFIG_EXPERIMENTAL" = "y" ]; then
-	 bool '    Philips SAA9730 Ethernet support (EXPERIMENTAL)' CONFIG_LAN_SAA9730
+      if [ "$CONFIG_EXPERIMENTAL" = "y" -a "$CONFIG_MIPS" = "y" ]; then
+	 dep_tristate '    Philips SAA9730 Ethernet support (EXPERIMENTAL)' CONFIG_LAN_SAA9730 $CONFIG_MIPS
       fi
    fi
    bool '  Pocket and portable adapters' CONFIG_NET_POCKET

-
To unsubscribe from this list: send the line "unsubscribe linux-kernel" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html
Please read the FAQ at  http://www.tux.org/lkml/
